examples/k3s/README.md

```diff
@@ -16,22 +31,35 @@ Or they all can be run with `make`:
 ```bash
 $ make
 ```
+K3S binary is already in the instances.
 
-You can then shell into any node, and check the status of K3S.
+### Upload K3S starter script and flux job submit script to ALL the nodes
 
 ```bash
-$ ssh -o 'IdentitiesOnly yes' -i "mykey.pem" rocky@ec2-xx-xxx-xx-xxx.compute-1.amazonaws.com
+$ scp -i "mykey.pem" k3s_starter.sh rocky@ec2-xx-xxx-xx-xxx.compute-1.amazonaws.com
```

Comment:
Any reason to use this strategy vs. adding a git clone in the setup script (so the user doesn't need to do it manually?)

Reply:
I also keep all the files in the directory-git clone should work. In doing this, my guess was if anyone wants to upload only these files to instances from there local home directory, they can do that?

Reply:
That's a great point! To make the user life easier, maybe it could be a combined approach - have the git clone to prepare the "production" (or last tested files) and then have a comment here of how to upload them, if needed. Since we are still in development I'm good if you want to keep as is and not do any kind of git clone yet.